### PR TITLE
Move logging setup outside lifespan and db initialisation logic

### PR DESCRIPTION
**Pull request by northwestwitch**
## Description
It's not a fix for #308, but setting up logging should be outside the function for the database setup 
- Moved logging setup out of app lifespan and db initialisation logic

### How to test
- [x]  Deploy on stage and make sure that the app starts and prints logging messages

## Review
- [x] Tests executed by CR
- [x] "Merge and deploy" approved by HS

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: src/chanjo2/main.py [hunk 1]
```diff
@@ -56,15 +65,6 @@ def configure_static(app):
 
 
 async def startup_db():
-    # Configure logging
```
Comment (northwestwitch) [inline src/chanjo2/main.py:59]: This was a bad place for the logging setup anyway, being a function that starts the database

**Comment by northwestwitch** [comment]
Tested on stage: 🆗 

image

File: src/chanjo2/main.py [hunk 2]
```diff
@@ -23,6 +23,15 @@
...
 
+# Configure logging
+console_formatter = uvicorn.logging.ColourizedFormatter(
+    "{levelprefix} {asctime} : {message}", style="{", use_colors=True
+)
+if LOG.handlers:
+    LOG.handlers[0].setFormatter(console_formatter)
+else:
+    logging.basicConfig()
+
```
Comment (henrikstranneheim) [inline src/chanjo2/main.py:35]: Would be nice to have this in a function in a separate module from main to keep it as free from logic as possible.

File: src/chanjo2/main.py [hunk 3]
```diff
@@ -23,6 +23,15 @@
     (overview.router, "overview"),
 ]
 
+# Configure logging
+console_formatter = uvicorn.logging.ColourizedFormatter(
```
Comment (henrikstranneheim) [inline src/chanjo2/main.py:27]: Is there an informative type hint for this?

File: src/chanjo2/main.py [hunk 4]
```diff
@@ -23,6 +23,15 @@
     (overview.router, "overview"),
 ]
 
+# Configure logging
```
Comment (henrikstranneheim) [inline src/chanjo2/main.py:26]:
```suggestion
```
No need if placed in a function.

**Review by henrikstranneheim — APPROVED** [review]
Good catch, 👍

File: src/chanjo2/main.py (hunk 2 above)
Comment (northwestwitch) [reply src/chanjo2/main.py]: Done, good suggestion! 👍🏻

File: src/chanjo2/main.py (hunk 3 above)
Comment (northwestwitch) [reply src/chanjo2/main.py]: it's an instance of the class uvicorn.logging.ColourizedFormatter. I think it's redundant to write it down

**Pull request merged**